digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/dao/RoadwayChangesDAOSpec.scala

```diff
@@ -387,4 +387,54 @@ class RoadwayChangesDAOSpec extends FunSuite with Matchers {
       roadNames.head should equal("Turku-Oulu")
     }
   }
+
+  test("When fetching type termination road address change info for road address browser then return the termination change info") {
+    runWithRollback {
+      val roadNameId = Sequences.nextRoadNameId
+      val projectId = Sequences.nextViiteProjectId
+      val roadwayChangeId = 10L
+      val changeTypeTerminated = 5L
+      val roadNumber = 8L
+      val roadPartNumber = 219L
+      val ely = 10L
+      val adminClass = 1
+      val discontinuity = 1L
+      val trackCombined = 0L
+      val projectStartDate = DateTime.parse("2022-07-01")
+      val terminatedRoadNameEndDate = DateTime.parse("2022-06-30") // one day earlier than the termination day of the road
+
+      sqlu"""insert into ROADWAY_CHANGES(project_id,change_type,old_road_number,old_road_part_number,old_track,old_start_addr_m,old_end_addr_m,
+                                            old_discontinuity,old_administrative_class,old_ely,new_discontinuity,new_ely,new_administrative_class,roadway_change_id)
+                                  values($projectId,$changeTypeTerminated,$roadNumber,$roadPartNumber,$trackCombined,0,1000,
+                                         $discontinuity,$adminClass,$ely,$discontinuity,$ely,$adminClass,$roadwayChangeId)
+          """.execute
+
+      // create terminated road name and history road name for road number 8
+      RoadNameDAO.create(
+        Seq(
+          RoadName(roadNameId, roadNumber, "Turku-Oulu", Some(DateTime.parse("1996-01-01")),Some(terminatedRoadNameEndDate),Some(DateTime.parse("2006-01-17")),None,"test"),
```

Comment:
Nillittäjä jää jumiin ku vanhempi tiennimi luodaan myöhemmin.
Rivien vaihto toisinpäin, ja id:t historiajärjestykessä, niin sit ei lukijaa ihmetytä